discord_api.py

```diff
@@ -124,11 +126,13 @@ def help(message):
 	if author_restrict == True:
 		msg += '!type <mensagem> - Envia uma mensagem para rodapé (só os users autorizados podem adicionar mensagens)\n'
```

Comment:
Please create a function to return a string with the messages, rather than duplicating them all over.

Reply:
Fixed